scss/detail.scss

```diff
@@ -25,7 +39,7 @@
   background: url('images/badge.png') no-repeat;
   content: '';
   height: 120px;
-  left: 250px;
+  left: 500px;
```

Comment:
Same here use position relative on .book-cover-row and take it like reference for absolute position.